alpaca_trade_api/rest.py

```diff
@@ -884,6 +898,43 @@ def get_crypto_snapshot(self, symbol: str, exchange: str) -> SnapshotV2:
                              api_version='v1beta1')
         return self.response_wrapper(resp, SnapshotV2)
 
+    def get_news_iter(self,
+                      symbol: Union[str, List[str]],
+                      start: Optional[str] = None,
+                      end: Optional[str] = None,
+                      limit: int = None,
```

Comment:
The limit handling in `_data_get` is incorrect unfortunately, because `DATA_V2_MAX_LIMIT` is not the actual max limit for the news API (it's 50 instead).

Reply:
Added `NEWS_MAX_LIMIT` and also exposing `total_limit` and `no_total_limit` parameters to `get_news()`